src/main/java/emissary/util/search/ByteTokenizer.java

```diff
@@ -19,7 +19,7 @@
  * characters in the range 0 - 255. In other words the ISO8859-1 encoding is used to match the delimiters to the byte
  * array.
  */
-public class ByteTokenizer implements Enumeration<String> {
+public class ByteTokenizer implements Iterator<String> {
```

Comment:
Maybe a separate pr for this??? May have downstream implications?

Reply:
I'm cool with reverting the changes made to this classes if that the right move. Would it require a different ticket too?

Reply:
> I'm cool with reverting the changes made to this classes if that the right move. Would it require a different ticket too?

Reverting wouldn't require a new issue, but the follow-on implementation would